src/client/components/List/index.js

```diff
@@ -10,11 +11,15 @@ export type Props = {
   pageSize: number,
   total: number,
   activePage: number,
-  children?: number | string | React.Element <*> | Array < any >,
-  onPageSelect?: Function,
+  children?: number | string | React.Element<*> | Array < any >,
+  onPageSelect ?: Function,
```

Comment:
This sudden space looks weird to me especially considering  `children?` just above it.